src/main/resources/sql/selectDiscreteGroundWater.sql

```diff
@@ -5,7 +5,7 @@ select
     d.measurement_source_code,
     d.measuring_agency_code,
     d.level_accuracy_code,
-    d.site_status_code,
+    d.result_measure_qualifiers::varchar,
```

Comment:
I thought about trying to select a single value from the JSON array right in the SQL, but I think it makes sense to use the ordering provided by the enum, which would be really verbose to do in the SQL.  It also makes it easier to see the logic.  
Note:  The actual preference order of qualifier codes is not specified, but I think it at least needs to be repeatable.  PostGres JSONB does not ensure an order to its values.

Reply:
AQTS probably does not guaranty an order either so I think we are ok with JSONB lack of order.